contracts/messaging/MessageBus.sol

```diff
@@ -10,5 +10,8 @@ contract MessageBus is MessageBusSender, MessageBusReceiver {
     constructor(address _gasFeePricing, address _authVerifier)
         MessageBusSender(_gasFeePricing)
         MessageBusReceiver(_authVerifier)
-    {}
+    {
+        // silence linter without generating bytecode
```

Comment:
any reason to not just // solhint-disable-next-line here?

Reply:
No special reason. `this;` just looks a bit cleaner, while serving exactly the same role.